src/main/java/io/antmedia/rest/PlaylistRestServiceV2.java

```diff
@@ -0,0 +1,215 @@
+import io.swagger.annotations.SwaggerDefinition;
+
+@Api(value = "Playlist Rest Service")
+@SwaggerDefinition(
+		info = @Info(
+				description = "Ant Media Server REST API Reference",
+				version = "V2.0",
+				title = "Ant Media Server REST API Reference",
+				contact = @Contact(name = "Ant Media Info", email = "contact@antmedia.io", url = "https://antmedia.io"),
+				license = @License(name = "Apache 2.0", url = "http://www.apache.org")),
+		consumes = {"application/json"},
+		produces = {"application/json"},
+		schemes = {SwaggerDefinition.Scheme.HTTP, SwaggerDefinition.Scheme.HTTPS},
+		externalDocs = @ExternalDocs(value = "External Docs", url = "https://antmedia.io"),
+		basePath = "/v2/Playlist"
+		)
+@Component
+@Path("/v2/playlists")
+public class PlaylistRestServiceV2 extends RestServiceBase{
+
+
+	@ApiOperation(value = "Playlist list from database", response = Playlist.class)
+	@GET
+	@Path("/{playlistId}")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Playlist getPlaylist(@ApiParam(value = "id of the Playlist", required = true) @PathParam("playlistId") String playlistId) {
+		Playlist playlist = null;
+		if (playlistId != null) {
+			playlist = getDataStore().getPlaylist(playlistId);
+		}
+		if (playlistId == null) {
+			playlist = new Playlist();
+		}
+		return playlist;
+	}
+
+	@ApiOperation(value = "Playlist list from database", response = Result.class)
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/{playlistId}/stop")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Result stopPlaylist(@ApiParam(value = "id of the Playlist", required = true) @PathParam("playlistId") String playlistId) {
+		Result result = new Result(false);
+
+		//Check playlistId is not null
+		if(playlistId != null) {
+
+			// Get playlist from Datastore
+			Playlist playlist = getDataStore().getPlaylist(playlistId);
+			// Get current broadcast from playlist
+			Broadcast broadcast = playlist.getBroadcastItemList().get(playlist.getCurrentPlayIndex());
+
+			if(!broadcast.getStreamId().isEmpty() && broadcast.getStreamId() != null) {
+
+				result = getApplication().stopStreaming(broadcast);
+
+				playlist.setPlaylistStatus(AntMediaApplicationAdapter.BROADCAST_STATUS_FINISHED);
+				getDataStore().editPlaylist(playlistId, playlist);
+			}
+		}
+
+		return result;
+	}
+
+	@ApiOperation(value = "Playlist list from database", response = Result.class)
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/{playlistId}/start")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Result startPlaylist(@ApiParam(value = "id of the Playlist", required = true) @PathParam("playlistId") String playlistId) {
+		Result result = new Result(false);
+
+		//Check playlistId is not null
+		if(playlistId != null) {
+			// Get playlist from Datastore
+			Playlist playlist = getDataStore().getPlaylist(playlistId);
+			// Get current broadcast from playlist
+			Broadcast broadcast = playlist.getBroadcastItemList().get(playlist.getCurrentPlayIndex());
```

Comment:
It may throw NullPointerException if play list id is not found in the db?

We need to have test code for this cases as well.